public/app/plugins/datasource/loki/querybuilder/components/LokiQueryEditorSelector.test.tsx

```diff
@@ -125,29 +132,30 @@ describe('LokiQueryEditorSelector', () => {
     });
   });
 
-  // it('parses query when changing to builder mode', async () => {
-  //   const { rerender } = renderWithProps({
-  //     refId: 'A',
-  //     expr: 'rate(test_metric{instance="host.docker.internal:3000"}[$__interval])',
-  //     editorMode: QueryEditorMode.Code,
-  //   });
-  //   switchToMode(QueryEditorMode.Builder);
-  //   rerender(
-  //     <PromQueryEditorSelector
-  //       {...defaultProps}
-  //       query={{
-  //         refId: 'A',
-  //         expr: 'rate(test_metric{instance="host.docker.internal:3000"}[$__interval])',
-  //         editorMode: QueryEditorMode.Builder,
-  //       }}
-  //     />
-  //   );
-
-  //   await screen.findByText('test_metric');
-  //   expect(screen.getByText('host.docker.internal:3000')).toBeInTheDocument();
-  //   expect(screen.getByText('Rate')).toBeInTheDocument();
-  //   expect(screen.getByText('$__interval')).toBeInTheDocument();
-  // });
+  it('parses query when changing to builder mode', async () => {
+    jest.spyOn(console, 'error').mockImplementation();
```

Comment:
Why is this here?

Reply:
Seems like there was unchecked async setState which triggered an error, so fixed that and this is not needed anymore